### ISSUE TITLE
BE.9 with 244 nt deletion in ORF7a beign designeted as BA.5.3.1

### ISSUE THREAD

**Issue by tiagofandreis** [cov-lineages/pango-designation#1430]
Some BE.9s sequences are being designated as BA.5.3.1, probably due to a 244 nt deletion in ORF7a (position 27508-27751) which were not reported in the original BE.9 sequence.

Here is an example in ORF7a:
![image](https://user-images.githubusercontent.com/120387016/207062665-161a21d6-53c7-4860-a106-ff6a975faa9f.png)

Mutations in S gene for the same samples. All expected mutation in BE.9 are present in both (S:Y144del; S:K444T; S:N460K; ORF1a:V84del; ORF1a:M85del)
![image](https://user-images.githubusercontent.com/120387016/207063617-932f92aa-3a86-47e3-be92-05577cb424f0.png)

**Comment by AngieHinrichs** [comment]
Hi @tiagofandreis, the sequence names are truncated in the nextclade views above - can you provide some complete sequence names?  Thanks.

**Comment by tiagofandreis** [comment]
Hi @AngieHinrichs ! Sorry about that. Here are the sequences that I used as an example:
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205713/2022|EPI_ISL_15603660|2022-10-14
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205714/2022|EPI_ISL_15603386|2022-10-14

It's important to note that the nucleotide changes that lead to S:444T and S:N460K mutations in BE.9 are A22893C and T22942A. Both of them are present in the given examples. 

Regarding the deletion in ORF7a, seems too big for me to be real. What's your thoughts?

Here is a list of sequences with the same issue that I found in GISAID:
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205854/2022\|EPI_ISL_15603380\|2022-10-18
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205878/2022\|EPI_ISL_15603381\|2022-10-19
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205871/2022\|EPI_ISL_15603382\|2022-10-19
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205850/2022\|EPI_ISL_15603383\|2022-10-18
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205801/2022\|EPI_ISL_15603384\|2022-10-17
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205752/2022\|EPI_ISL_15603385\|2022-10-16
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205714/2022\|EPI_ISL_15603386\|2022-10-14
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205892/2022\|EPI_ISL_15603387\|2022-10-19
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205934/2022\|EPI_ISL_15808901\|2022-10-21
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205935/2022\|EPI_ISL_15808902\|2022-10-22
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205936/2022\|EPI_ISL_15808903\|2022-10-22
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205937/2022\|EPI_ISL_15808904\|2022-10-22
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205938/2022\|EPI_ISL_15808905\|2022-10-22
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205939/2022\|EPI_ISL_15808906\|2022-10-23
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205940/2022\|EPI_ISL_15808907\|2022-10-23
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205941/2022\|EPI_ISL_15808908\|2022-10-23
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205942/2022\|EPI_ISL_15808909\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205943/2022\|EPI_ISL_15808910\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205944/2022\|EPI_ISL_15808911\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205945/2022\|EPI_ISL_15808912\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205946/2022\|EPI_ISL_15808913\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205947/2022\|EPI_ISL_15808914\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205948/2022\|EPI_ISL_15808915\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205949/2022\|EPI_ISL_15808916\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205950/2022\|EPI_ISL_15808917\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205951/2022\|EPI_ISL_15808918\|2022-10-24
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205952/2022\|EPI_ISL_15808919\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205953/2022\|EPI_ISL_15808920\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205954/2022\|EPI_ISL_15808921\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205955/2022\|EPI_ISL_15808922\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205956/2022\|EPI_ISL_15808923\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205957/2022\|EPI_ISL_15808924\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205958/2022\|EPI_ISL_15808925\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205959/2022\|EPI_ISL_15808926\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205960/2022\|EPI_ISL_15808927\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205961/2022\|EPI_ISL_15808928\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205962/2022\|EPI_ISL_15808929\|2022-10-25
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205963/2022\|EPI_ISL_15808930\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205964/2022\|EPI_ISL_15808931\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205965/2022\|EPI_ISL_15808932\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205966/2022\|EPI_ISL_15808933\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205967/2022\|EPI_ISL_15808934\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205968/2022\|EPI_ISL_15808935\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205969/2022\|EPI_ISL_15808936\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205970/2022\|EPI_ISL_15808937\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205971/2022\|EPI_ISL_15808938\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205972/2022\|EPI_ISL_15808939\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205973/2022\|EPI_ISL_15808940\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205974/2022\|EPI_ISL_15808941\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205975/2022\|EPI_ISL_15808942\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205976/2022\|EPI_ISL_15808943\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205977/2022\|EPI_ISL_15808944\|2022-10-26
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205978/2022\|EPI_ISL_15808945\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205979/2022\|EPI_ISL_15808946\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205980/2022\|EPI_ISL_15808947\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205981/2022\|EPI_ISL_15808948\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205982/2022\|EPI_ISL_15808949\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205983/2022\|EPI_ISL_15808950\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205984/2022\|EPI_ISL_15808951\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205985/2022\|EPI_ISL_15808952\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205986/2022\|EPI_ISL_15808953\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205987/2022\|EPI_ISL_15808954\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205988/2022\|EPI_ISL_15808955\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205989/2022\|EPI_ISL_15808956\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205990/2022\|EPI_ISL_15808957\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205991/2022\|EPI_ISL_15808958\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205992/2022\|EPI_ISL_15808959\|2022-10-27
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205993/2022\|EPI_ISL_15808960\|2022-10-28
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205994/2022\|EPI_ISL_15808961\|2022-10-28
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205995/2022\|EPI_ISL_15808962\|2022-10-28
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205996/2022\|EPI_ISL_15808963\|2022-10-28
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205997/2022\|EPI_ISL_15808964\|2022-10-28
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205998/2022\|EPI_ISL_15808965\|2022-10-31
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2205999/2022\|EPI_ISL_15808966\|2022-10-31
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2206000/2022\|EPI_ISL_15808967\|2022-10-31
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2206001/2022\|EPI_ISL_15808968\|2022-10-31
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2206002/2022\|EPI_ISL_15808969\|2022-10-31
hCoV-19/Brazil/AM-FIOCRUZ-ILMD2206003/2022\|EPI_ISL_15808970\|2022-11-04
hCoV-19/Brazil/RS-GAIHN-BRA-01-0021-F/2022\|EPI_ISL_16014612\|2022-11-18
hCoV-19/Brazil/RS-GAIHN-BRA-01-0038-F/2022\|EPI_ISL_16014613\|2022-11-21

**Comment by AngieHinrichs** [comment]
Thanks!  Those sequences are all in BE.9 in the UShER tree and are called BE.9 by pangolin using the latest pangolin-data release ([v1.17](https://github.com/cov-lineages/pangolin-data/releases/tag/v1.17)) in the default usher mode.  (It looks like pangoLEARN v1.17 is calling most of them BE.5.)  Are you seeing the BA.5.3.1 assignments only from nextclade?  If so, are you using nextclade only on the website (https://clades.nextstrain.org) or also on the command line?  

If you are using pangolin on the command line, and `pangolin --all-versions` shows `pangolin-assignment: 1.16` or earlier, then try `pangolin --update` to get the v1.17 data release which can assign BE.9.

If you are seeing the BA.5.3.1 assignments on GISAID, it's probably because the v1.17 data release just happened yesterday and GISAID is still in the process of updating assignments.

**Comment by silcn** [comment]
@AngieHinrichs I think this 244nt deletion is real and present in most if not all BE.9. 

The WT sequence is ...CTTCTGGA**A**...**A**GACAGAAT... where the nucleotides in bold are 27508 and 27749.

The Chilean sequences have ...CTTCTGGACAGAAT... with 27508-27751del and no other nucleotide mutations. I think this is correct. Note the "GGAC" that has been created in the middle - it's important.

The Brazilian sequences instead have A27508C, then NNNs from 27509-27748, then A27749G. What this looks like is ...CTTCTGGA**C**NNN...NNN**G**GACAGAAT... It seems the huge deletion hasn't been dealt with properly. In its place are 240 NNNs, along with a duplication of GGAC, which has probably happened because it matches up well with both the GGAA in the reference sequence around the start of the deletion (27505-27508) and the AGAC around the end (27749-27752).

In other words, the substitutions A27508C and A27749G are bogus and should be removed from the Nextclade definition of BE.9.

**Comment by FedeGueli** [comment]
Thx @silcn, mistery solved. i think covspectrum got it right since the start when it did find little with the two mutations .

**Comment by corneliusroemer** [comment]
This will be fixed in the next Nextclade dataset release, see https://github.com/neherlab/nextclade_data_workflows/commit/0ca74c899049a4c488e66ad3083b6f9922a348f0

If you find similar spurious mutations in Nextclade datasets let me know and I'll fix it. Thanks @silcn!

**Comment by corneliusroemer** [comment]
I'll close this issue as it's mostly Nextclade specific and fixed in Nextclade. Thanks for reporting @tiagofandreis!

**Comment by tiagofandreis** [comment]
Thank you all! And yes @AngieHinrichs , Pango 1.17 is calling all the sequences as BE.9. I had not updated the database at that time.  Thanks for checking in the Nextclade dataset @corneliusroemer
